.gitignore

```diff
@@ -0,0 +1,15 @@
+## Idea Folder
```

Comment:
Please add all the files under the `.idea` directory, as you are pushing files which are not needed

Reply:
done